packages/calendar/src/features/date-range-text-input/DateRangeDualTextInput.tsx

```diff
@@ -160,9 +160,15 @@ export const DateRangeDualTextInput: React.FC<Props> = ({
   }, [setCurrentPanel, showCalendar]);
 
   const onClickCalendarButton = useCallback(() => {
-    setCurrentPanel("calendar");
-    showCalendar();
-  }, [setCurrentPanel, showCalendar]);
+    if (focusedInput === "startDate" && startDateInputRef.current) {
+      startDateInputRef.current.focus();
```

Comment:
```suggestion
    if (focusedInput === "startDate") {
      startDateInputRef.current?.focus();
```

Reply:
Nope. That would not trigger the else clause.